### PR TITLE
Copyedits to ts/logging.md

### PR DESCRIPTION
**Pull request by lorensr**

### CODE REVIEW

File: docs/typescript/logging.md [hunk 1]
```diff
@@ -155,18 +155,20 @@ await Core.install({ logger });
 
 We are in the process of building out our SDK metrics capabilities, for now please observe standard monitoring practices on your Workers in production (CPU, memory utilization, health checks).
```
Comment (djmagee) [inline docs/typescript/logging.md:156]:
```suggestion
We are in the process of building out our SDK metrics capabilities. For now, please observe standard monitoring practices on your Workers in production (CPU, memory utilization, health checks).
```

File: docs/typescript/logging.md [hunk 2]
```diff
@@ -155,18 +155,20 @@ await Core.install({ logger });
...
+The [`interceptors-opentelemetry`](https://github.com/temporalio/samples-typescript/tree/main/interceptors-opentelemetry) sample shows how to use the SDK's built-in OpenTelemetry tracing to trace everything from starting a Workflow to Workflow Execution to running an Activity from that Workflow.
 
-The built-in tracing uses [protobuf message headers](https://github.com/temporalio/api/blob/b2b8ae6592a8730dd5be6d90569d1aea84e1712f/temporal/api/workflowservice/v1/request_response.proto#L161) to propagate the tracing information from the client to the Workflow and from the Workflow to its successors (when continued as new), children and Activities.
+The built-in tracing uses protobuf message headers (like [this one](https://github.com/temporalio/api/blob/b2b8ae6592a8730dd5be6d90569d1aea84e1712f/temporal/api/workflowservice/v1/request_response.proto#L161) when starting a Workflow) to propagate the tracing information from the client to the Workflow and from the Workflow to its successors (when Continued As New), children, and Activities.
 All of these executions will be linked with a single trace ID and will have the proper parent->child span relation.
```
Comment (djmagee) [inline docs/typescript/logging.md:163]:
```suggestion
All of these executions are linked with a single trace ID and have the proper parent->child span relation.
```

File: docs/typescript/logging.md [hunk 3]
```diff
@@ -155,18 +155,20 @@ await Core.install({ logger });
...
 
-Tracing is compatible between the different Temporal SDKs as long as compatible context propagators are used.
+Tracing is compatible between the different Temporal SDKs as long as compatible [context propagators](https://opentelemetry.lightstep.com/core-concepts/context-propagation/) are used.
```
Comment (djmagee) [inline docs/typescript/logging.md:165]:
```suggestion
Tracing is compatible between different Temporal SDKs as long as compatible [context propagators](https://opentelemetry.lightstep.com/core-concepts/context-propagation/) are used.
```

File: docs/typescript/logging.md [hunk 4]
```diff
@@ -182,16 +184,14 @@ To extend default and include the jaeger propagator, follow these steps:
...
+        new W3CTraceContextPropagator(),
+        new W3CBaggagePropagator(),
+        new JaegerPropagator(),
+      ],
+    })
   );
   ```
 
-- Similarly you may customize the OpenTelemetry Node SDK propagators by following the instructions on [this page](https://github.com/open-telemetry/opentelemetry-js/tree/main/experimental/packages/opentelemetry-sdk-node#initialize-the-sdk)
+- Similarly, you may customize the OpenTelemetry Node SDK propagators by following the instructions on [this page](https://github.com/open-telemetry/opentelemetry-js/tree/main/experimental/packages/opentelemetry-sdk-node#initialize-the-sdk).
```
Comment (djmagee) [inline docs/typescript/logging.md:197]:
```suggestion
Similarly, you can customize the propagators of the OpenTelemetry SDK for Node.js by following the instructions in the [Initialize the SDK](https://github.com/open-telemetry/opentelemetry-js/tree/main/experimental/packages/opentelemetry-sdk-node#initialize-the-sdk) section of the README for the SDK.
```

Comment (lorensr) [reply docs/typescript/logging.md:197]:
```suggestion
Similarly, you can customize the OpenTelemetry `NodeSDK` propagators by following the instructions in the [Initialize the SDK](https://github.com/open-telemetry/opentelemetry-js/tree/main/experimental/packages/opentelemetry-sdk-node#initialize-the-sdk) section of the README.
```

Comment (flossypurse) [reply docs/typescript/logging.md:197]: Node SDK -> TypeScript SDK ?

Comment (lorensr) [reply docs/typescript/logging.md:197]: It's an opentelemetry library, not ours

**Review by sw-yx — APPROVED** [review]

**Review by flossypurse — APPROVED** [review]

**Pull request merged**